apps/web/pages/event-types/[type]/index.tsx

```diff
@@ -326,6 +315,18 @@ const EventTypePage = (props: inferSSRProps<typeof getServerSideProps>) => {
   );
 };
 
+const EventTypePageWrapper = (props: inferSSRProps<typeof getServerSideProps>) => {
```

Comment:
this wrapper makes sure EventTypePage is loaded only when data is not null or undefined else we would have to use too much of optional chaining and && operator. ( suggested by  hariom)